packages/api/es/indexer.js

```diff
@@ -336,6 +341,44 @@ async function reingest(g) {
   return gObj.reingest(g);
 }
 
+const isCompletedExecutionMessage = (cumulusMessage) =>
```

Comment:
These are all just functions I extracted out of `handlePayload` to make that function easier to read.

Reply:
The only place that `isCompletedExecutionMessage` is used is inside of `buildExecutionRecord`, which is passed in a `cumulusMessage`.  It felt like it made `buildExectionRecord` more readable this way.